hwilib/devices/trezor.py

```diff
@@ -95,6 +97,39 @@ def parse_multisig(script):
     multisig = proto.MultisigRedeemScriptType(m=m, signatures=[b''] * n, pubkeys=pubkeys)
     return (True, multisig)
 
+# Parses the PSBT_GLOBAL_XPUB fields of a PSBT as multisig pubkeys
+def parse_multisig_xpubs(tx, psbt_in_out, multisig):
+    try:
+        old_pubs = [k.node.public_key for k in multisig.pubkeys]
+        xpubs = [xpub for xpub in tx.unknown.keys() if xpub.startswith(b"\x01")]
+        derivations = [tx.unknown[xpub] for xpub in xpubs]
+        # unpack
+        derivations = [[value[4 * i:4 * i + 4] for i in range(0, len(value) // 4)] for value in derivations]
+        derivations = [[int.from_bytes(v, 'little') for v in der] for der in derivations]
+        new_pubs = []
+        for pub in old_pubs:
+            # derivation
+            der = list(psbt_in_out.hd_keypaths[pub])
+            for i, derivation in py_enumerate(derivations):
+                if der[0] == derivation[0]:
+                    idx = i
+                    for i in range(derivation):
```

Comment:
`range(len(derivation))`

Reply:
Opps... right, added. (shouldn't do changes of the last moment when submitting PRs...)